.jenkins/continuous.groovy

```diff
@@ -275,6 +275,66 @@ pipeline {
                     }
                 }
 
+                stage('gcc') {
+                    agent {
+                        dockerfile {
+                            filename "Dockerfile"
+                            dir "docker"
+                            additionalBuildArgs '--build-arg BASE=gcc:12.2.0 --build-arg KOKKOS_OPTIONS="-DCMAKE_CXX_EXTENSIONS=OFF -DKokkos_ENABLE_OPENMP=ON -DCMAKE_CXX_COMPILER=g++ -DCMAKE_PREFIX_PATH=/usr/lib/x86_64-linux-gnu"'
```

Comment:
Why do you need to add `CMAKE_CXX_EXTENSIONS=OFF` and specify `CMAKE_PREFIX_PATH`?

Reply:
I just copied from the clang stage